test/test_helper.rb

```diff
@@ -174,6 +174,18 @@ def read_file(filename)
 
 end
 
+# Find Chromium and chromedriver in serveral common locations
+Selenium::WebDriver::Chrome.path = ENV["CHROMIUM_BIN"] || %w[
+  /usr/bin/chromium-browser
+  /Applications/Chromium.app/Contents/MacOS/Chromium
+  /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome
+].find { |path| File.executable?(path) }
+Selenium::WebDriver::Chrome.driver_path = ENV["CHROMEDRIVER_PATH"] || %w[
+  /usr/bin/chromedriver
+  /usr/lib/chromium-browser/chromedriver
+  /usr/local/bin/chromedriver
+].find { |path| File.executable?(path) }
+
```

Comment:
Had to remove this. Broke it for me as it wouldn't find my chromedriver that lives in a different bin directory (but still on my $PATH). There has to be a documentation on Rails Guides for installing all the crap. Guides only have a bit about testing, but nothing about how to get it all running in the first place.